docs/Modding/mapmaking/Objects.md

```diff
@@ -20,3 +20,18 @@ For example this matches anything with "fire" in the name or Road Line Cap #1:
     Cap #1 Road Line
 
 Recursive usage of filters is supported, so multiple favorite searches can be nested, or other filter types e.g. "Tunnel mb:core" includes tunnels from the vanilla objects.
+
+## Editor Asset Redirectors
+
+If many objects need to be replaced on a map the old object guid can be redirected to a new guid rather than manually replacing them. This works similarly to the automatic holiday object replacements, but applies while loading a map in the editor, and changes are kept when the map is saved.
+
+The game looks for a file named "EditorAssetRedirectors.txt" in the Unturned folder. Empty lines and lines starting with "//" or "#" (comments) are ignored. The .txt file extension was chosen because it is the notepad default. Each line contains two guids separated by an arrow "->".
```

Comment:
```suggestion
The game looks for a file named "EditorAssetRedirectors.txt" in the Unturned folder. Empty lines and lines starting with `//` or `#` (comments) are ignored. The .txt file extension was chosen because it is the notepad default. Each line contains two guids separated by an arrow `->`.
```
Would code blocks work better here?

Reply:
Sure I guess, Nelson himself did not include the code blocks, as this was partially taken from U3 Docs, but I'll sign off on it. Thanks for catching that.